contracts/math/log.ral

```diff
@@ -129,15 +129,15 @@ Abstract Contract Log() extends Decimal() {
 
         let mut accurateTick = 0i
         if (log2Sign) {
-            let fartherTickSqrtPriceDecimal = sqrtPriceFromTick(fartherTick)
-            if (sqrtPrice >= fartherTickSqrtPriceDecimal) {
+            let fartherTickSqrtPriceDecimal = sqrtPriceFromTick(fartherTick).v
```

Comment:
It would look better if you extract he `.v` later, since the naming does not match. The variable is storying underlying type of `SqrtPrice` not directly `SqrtPrice`